main/test/ca/uwaterloo/flix/library/TestFile.flix

```diff
@@ -95,4 +95,12 @@ namespace TestFile {
     def testIsExecutable(): Bool & Impure =
         isExecutable("./fakeREADME.md") == Ok(false)
 
+    /////////////////////////////////////////////////////////////////////////////
+    // readLines                                                               //
```

Comment:
can we test the other read* functions? :)

Reply:
I was working on it. There is just a problem with readWith we should maybe get fixed